docs/img/api_coverage_comparison.svg

```diff
@@ -0,0 +1,222 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
This is rendering a little weird for me: 
![image](https://user-images.githubusercontent.com/10732128/145243207-a6aedaf7-0235-4d11-bca9-f0e319e7231f.png)

Reply:
Updated the SVG to fix this issue. It was displaying weirdly on Safari but now it works on both Safari and Chrome.